src/Services/Catalog/Catalog.Domain/Abstractions/ICategoryRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Catalog.Domain.Entities;
+
+namespace Catalog.Domain.Abstractions;
+
+public interface ICategoryRepository
```

Comment:
interfaces of repositories should be located on the same level  with repositories

Reply:
in infrastructure level?